website/blog/2018-08-24-7.0.0.md

```diff
@@ -0,0 +1,442 @@
+---
+layout: post
+title:  "Babel 7 Released"
+author: Henry Zhu
+authorURL: https://twitter.com/left_pad
+date:   2018-08-24 18:00:00
+categories: announcements
+share_text: "Babel 7 Released"
+---
+
+After over 1.5 years, 4k commits, over 50 pre-releases, and a lot of help we are excited to announce the release of Babel 7. It's been almost [3 years](https://babeljs.io/blog/2015/10/29/6.0.0) since the release of Babel 6! There's a lot of moving parts so please bear with us in the first weeks of release.
+
+<!-- truncate -->
+
+If you appreciate the work we're doing on Babel, you can support me on [Patreon](https://www.patreon.com/henryzhu), ask your company to sponsor Babel on [Open Collective](https://opencollective.com/babel), and also get your company involved with Babel as part of your work. We'd appreciate the collective ownership of this vital project in the frontend community!
```

Comment:
I'm fine with that, but "If you appreciate the work we're doing [...], you can support me" sounds strange.

Reply:
Sure, I just moved this above because it's unlikely people will read through to the end (usually I put it as the last thing)